templates/nextflow-aurora-mysql.yaml

```diff
@@ -87,12 +86,12 @@ Resources:
           - - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
```

Comment:
[optional] to further simplify, you could just pass in this ARN as well.  Then this whole thing just becomes a reference to the parameter, `!Ref  AccountAdminArns`.

Reply:
Is there a way of obtaining the account ID in the Sceptre config (like `${AWS::AccountId}` in CFN)?

Reply:
I don't think so.